src/c_api/c_api.cc

```diff
@@ -972,6 +972,44 @@ int MXNDArrayCreateSparseEx(int storage_type,
       Context::Create(static_cast<Context::DeviceType>(dev_type), dev_id),
       delay_alloc != 0,
       dtype, aux_types, aux_shapes);
+}
+
+int MXNDArrayCreateSparseEx(int storage_type,
+                            const uint32_t *shape,
+                            uint32_t ndim,
```

Comment:
Can we also change the ndim type here from uint32_t to int? It looks strange to me why 32-bit version uses uint32_t while 64-bit version uses int.